lib/Doctrine/CouchDB/CouchDBClient.php

```diff
@@ -807,6 +807,29 @@ public function ensureFullCommit()
             throw HTTPException::fromResponse($path, $response);
         }
 
+        return $response->body;
+    }
+
+    /**
+     * Bulk get.
+     *
+     * @param array $docs
```

Comment:
What type are the elements in this array?

Reply:
```
[
    ['id' => 'docID-1', 'rev' => '3-rev'],
    ['id' => 'docID-2'],
]
```
rev key is optional.
https://docs.couchdb.org/en/stable/api/database/bulk-api.html#db-bulk-get

Reply:
Ok so
```suggestion
     * @param array<int, array<string, string>> $docs
```

or even
```suggestion
     * @param array<int, array{id: string, rev?: string}> $docs
```
(but I'd like another contributor chime in about that one, not sure we are allowed to use such advanced descriptions yet)